src/templates/pycontw-2020/contents/zh/about/privacy_policy.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}PyCon Taiwan 個人資料保護聲明{% endblock title %}
+
+{% block content %}
+
+<section class="pill-title alt">
+	<h1>PyCon Taiwan 個人資料保護聲明</h1>
+</section>
+<h5 style="text-align:right;">(CC BY-SA 3.0 TW)<br/>最後更新時間：2020-08-04</h5>
+<h4>個人資料保護政策</h4>
+
+<p>感謝您參與臺灣 Python 年會 (以下簡稱「PyCon TW」)。「PyCon TW 主辦團隊」(以下簡稱「我們」)
+	將依個人資料保護法之規定來使用、保管您於報名時所提供的個人資料，並維護您的隱私權。同時，我們在此對於您個人資料之使用蒐集將依下列聲明為利用、管理。</p>
+
+<h4>個人資料之使用目的</h4>
+
+<p>我們為求本活動順利進行及提供您相關之服務，蒐集並於使用目的範圍內利用您所提供之個人資料，<b>其法定特定目的為一三〇會議管理</b>。使用目的包含但不限於：</p>
+<ol>
+	　
+	<li>確認報名者與前來報到者之身分同一性</li>
+	　
+	<li>提供您的名牌</li>
+	　
+	<li>其他有助於活動順利進行之相關目的</li>
+</ol>
+
+<h4>個人資料蒐集方式、利用範圍</h4>
+
+<p>當您報名參與 PyCon TW 時，我們基於服務將會請您填寫服務所需之相關資料，而取得您的基本資料；我們僅將您的該些資料使用在您所需的個別服務上。</p>
+
+<h4>個人資料使用方式</h4>
+<p>我們將持續保存使用您提供的個人資料直至您提出停止使用或 PyCon TW 停止營運之日為止。我們<b>絕對不會</b>任意出售、交換、出租或以其他變相之方式，將您的個人資料揭露與其他團體或個人。惟有下列情形，我們會向第三者提供您的個人資料。</p>
+<ol>
+	　
+	<li>經過您的事前同意或授權允許時。</li>
+	　
+	<li>司法單位或其他主管機關經合法正式的程序要求時。</li>
+</ol>
```

Comment:
Excessive line break
![image](https://user-images.githubusercontent.com/24987826/89689079-2d518500-d936-11ea-8e67-55b324a2fa4d.png)

```suggestion
<ol>
	<li>經過您的事前同意或授權允許時。</li>
	<li>司法單位或其他主管機關經合法正式的程序要求時。</li>
</ol>
```